src/config.rs

```diff
@@ -19,15 +19,22 @@ pub fn global_config_path() -> String {
     path.to_str().unwrap().to_owned()
 }
 
+pub fn remove_trailing_slash(url: String) -> String {
+    let mut trimmed = url.clone();
+    if Some('/') == trimmed.chars().last() {
+        trimmed.pop();
+    }
+    trimmed.trim().to_owned()
+}
```

Comment:
In my hack in pdb for now I used `(clojure.string/replace "/x//foo//" #"/+$" "")`, i.e. it removes any number of trailing slashes - wonder if we care about that here, or if that should just be rejected.